docs/iceberg-rest-service.md

```diff
@@ -65,18 +65,19 @@ Please note that, it only takes affect in `gravitino.conf`, you don't need to sp
+| `gravitino.iceberg-rest.threadPoolWorkQueueSize` | The size of the queue in the thread pool used by Gravitino Iceberg REST catalog service.                                                                                                                                            | `100`                                                                        | No       | 0.2.0         |
+| `gravitino.iceberg-rest.stopTimeout`             | The amount of time in ms for the Gravitino Iceberg REST catalog service to stop gracefully. For more information, see `org.eclipse.jetty.server.Server#setStopTimeout`.                                                             | `30000`                                                                      | No       | 0.2.0         |
+| `gravitino.iceberg-rest.idleTimeout`             | The timeout in ms of idle connections.                                                                                                                                                                                              | `30000`                                                                      | No       | 0.2.0         |
+| `gravitino.iceberg-rest.requestHeaderSize`       | The maximum size of an HTTP request.                                                                                                                                                                                                | `131072`                                                                     | No       | 0.2.0         |
+| `gravitino.iceberg-rest.responseHeaderSize`      | The maximum size of an HTTP response.                                                                                                                                                                                               | `131072`                                                                     | No       | 0.2.0         |
+| `gravitino.iceberg-rest.customFilters`           | Comma-separated list of filter class names to apply to the APIs.                                                                                                                                                                    | (none)                                                                       | No       | 0.4.0         |
+| `gravitino.iceberg-rest.catalog-provider-impl`   | The implementation of IcebergTableOpsProvider defines how the Iceberg REST catalog server gets iceberg catalogs.                                                                                                                    | `org.apache.gravitino.iceberg.common.ops.ConfigIcebergTableOpsProvider`      | No       | 0.6.0         |
```

Comment:
the other config items are for HTTP servers, it's better to create a new section like `Multi catalog`, and provide an example about how to configuration multi catalogs.